pkg/producer/http_config.go

```diff
@@ -0,0 +1,37 @@
+package producer
+
+import (
+	"context"
+	"net/url"
+)
+
+// ProducerConfig holds configuration required to send Nexpose assets
+// to a queue via an HTTP Producer
+type ProducerConfig struct {
+	Endpoint string `description:"The scheme and host of a streaming appliance."`
+}
+
+// Name is used by the settings library and will add a "HTTPPRODUCER"
+// prefix to ProducerConfig environment variables
+func (c *ProducerConfig) Name() string {
+	return "HTTPProducer"
```

Comment:
nit: might want to add just `HTTPPRODUCER_ENDPOINT` env var to compose file as a placeholder

Reply:
😬 These files shouldn't have been committed in the first place. I'll remove these from the PR.